kyo-core/shared/src/main/scala/kyo/streams.scala

```diff
@@ -273,60 +238,61 @@ object Stream:
         s
 
     private object internal:
-        private val discard = new Handler[Const[Any], Streams[Any], Any]:
-            def resume[T, U: Flat, S](command: Any, k: T => U < (Streams[Any] & S)) =
+        private val discard = new Handler[Const[Chunk[Any]], Streams[Any], Any]:
+            def resume[T, U: Flat, S](command: Chunk[Any], k: T => U < (Streams[Any] & S)) =
                 Resume((), k(().asInstanceOf[T]))
 
-        def discardHandler[V]: Handler[Const[V], Streams[V], Any] =
-            discard.asInstanceOf[Handler[Const[V], Streams[V], Any]]
+        def discardHandler[V]: Handler[Const[Chunk[V]], Streams[V], Any] =
+            discard.asInstanceOf[Handler[Const[Chunk[V]], Streams[V], Any]]
     end internal
 
 end Stream
 
 class Streams[V] extends Effect[Streams[V]]:
-    type Command[T] = V
+    type Command[T] = Chunk[V]
```

Comment:
pre-existing: is `T` used?

Reply:
More or less, this effect suspends with a constant value so the the suspended type isn't directly related to the command. This allows returning a `Unit` in `Streams.emit*`, which then gets resumed in the handlers like in the previous comment. This mechanism isn't very type safe but I consider the core API still internal. We'll need to find ways to make it safer and more ergonomic to allow users to create new effects.